octobot_trading/enums.py

```diff
@@ -189,6 +190,34 @@ class ExchangePositionCCXTColumns(enum.Enum):
     PERCENTAGE = "percentage"
     SIDE = "side"
     INFO = "info"
+    
+class CCXTExchangeConfigMethods(enum.Enum):
+    GET_ORDER_DEFAULT = "get_order_default"
```

Comment:
I don't like hardcoding method's name, it's hard to maintain

Reply:
I changed it to method.__name__